src/styles/docs/styles.css

```diff
@@ -1,10 +1,14 @@
 /* layouts Grid Column */
-.customColumnClass {
-  border: 1px solid blue;
+.customColumnClass:nth-child(odd) {
+  background-color: #ddd;
```

Comment:
let me try, but I cannot see any advantage of using scss for documentation because this is only for overriding default document style.